### ISSUE TITLE
!!!!BUTTONS!!!!STYLES!!!!NEED!!!!HELP!!!!ASAP!!!!PLS.THX. 

### ISSUE THREAD

**Issue by zama0023** [zama0023/button-styles#1]
MarkBot is telling me i cant have any properties in any of my btn classes except of ".btn" 
Not sure where to place the properties so that it will stop sending errors and it can be functional. 
@thomasjbradley @thomasjbradley @thomasjbradley @thomasjbradley @thomasjbradley @thomasjbradley @thomasjbradley @thomasjbradley @thomasjbradley @thomasjbradley

**Comment by thomasjbradley** [comment]
Yep that's right! The `.btn` class is shared between all the buttons, so everything that is common, goes there. The other classes only have the couple properties that are different.

Look a this slide here to see get a hint on how to do it:
https://learn-the-web.algonquindesign.ca/courses/web-dev-1/link-patterns/#slide-5
